index.js

```diff
@@ -43,12 +43,13 @@ if (!YARNHOOK_BYPASS) {
     console.log("CMD:", CMD);
   }
 
-  if (lockfile !== null) {
+  if (lockfile) {
     // run a git diff on the lockfile
-    const output = execSync(`git diff HEAD@{1}..HEAD@{0} -- ${lockfile}`, {
-      cwd: gitDir,
-      encoding: "utf-8"
-    });
+    const { stdout: output } = execa.sync(
```

Comment:
Does it always return a string? The docs say the return value is same as [child_process.spawnSync](https://nodejs.org/api/child_process.html#child_process_child_process_spawnsync_command_args_options), which is documented as `<Buffer> | <string>`.

Reply:
I tested this and I think its length is never zero. I wonder how it decides to return either Buffer or string.

Reply:
Yep, I tested it and it seems to work fine.